kiwi/storage/subformat/vagrant_base.py

```diff
@@ -147,31 +159,30 @@ def store_to_result(self, result):
 
         :param object result: Instance of Result
         """
-        result.add(
-            key='disk_format_image',
-            filename=self.get_target_file_path_for_format(
-                self.image_format
-            ),
-            use_for_bundle=True,
-            compress=False,
-            shasum=True
-        )
+        if self.image_format:
```

Comment:
The same suggestion as for `create_image_format` applies here too.

Reply:
done, raising on improper setup